src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -94,4 +95,21 @@ public function without(string ...$keys): self
 
         return $clone;
     }
+
+    public function add(AbstractStimulusDto $stimulusDto): self
```

Comment:
Isn't a bit weird to have a method `add` that takes a `StimulusDto`? 
Shouldn't this method be named `withStimulus` or something like that?

Reply:
Probably :). But it looked a bit terrible when used:

```
{{ attributes.withStimulus(stimulus_controller(...)) }}
```

The 2x Stimulus. But, we could potentially "open" this `add()` method later to accept other types of arguments, if we think of something.

Reply:
Yeah, an array of key/values for example